jira.go

```diff
@@ -278,6 +300,8 @@ func openJiraTickets(flags flags, projectInfo jsn.Json, vulnsForJira map[string]
 	project[projectIdString] = ticketArray
 
 	if fullResponseDataAggregated == "" && !flags.optionalFlags.dryRun {
+		message := fmt.Sprintf("*** ERROR *** Request response from %s is empty\n", flags.mandatoryFlags.endpointAPI)
```

Comment:
`Received empty response from %s` is easier to see the reason, instead of having to look after a long URL